src/main/index.ts

```diff
@@ -106,10 +114,33 @@ app.on("ready", async () => {
     app.exit();
   }
 
+  // test proxy connection
+  try {
```

Comment:
Could we refactor this into a separate function?

Reply:
Refactored

Reply:
The try block?

Reply:
> The try block?

Don't get. What about it?

Reply:
I was expecting the whole try block to be refactored out

Reply:
Okay... I think it's now in line with other checks present in that file.

Reply:
I meant the whole try block but I'm fine with this (it was just a suggestion).